cypress/e2e/clue/branch/student_tests/drawing_tool_spec.js

```diff
@@ -123,7 +123,6 @@ context('Draw Tool Tile', function () {
           .trigger("mousemove", 200, 100)
           .trigger("mouseup", 200, 100);
         // For some reason the move isn't very accurate in cypress so often the final location off
-        drawToolTile.getRectangleDrawing().first().should("have.attr", "x").then(parseInt).and("within", 160, 220);
```

Comment:
This test is working or me locally. Can you add it back in?

Reply:
Done.